Dockerfile

```diff
@@ -3,40 +3,49 @@ FROM       debian:stretch
 
 ARG GHC_VERSION
 ARG STACK_VERSION
-ARG CABAL_VERSION
-ARG HAPPY_VERSION
-ARG ALEX_VERSION
```

Comment:
@telser Are cabal, happy and alex not needed because they are included in the ghc package on haskell.org?

Reply:
@qxjit IIRC, cabal yes, the other two picked up by stack and installed now.

Reply:
@telser Should we be using stack to install ghc rather than downloading deb packages?

Reply:
@qxjit I tried, but hit a wall. We should chat about it, because if it can work I think that'd be a nicer solution.